app/views/automated_tests/_test_support_file_upload.html.erb

```diff
@@ -22,23 +23,28 @@
           </span>
           <br/>
 
-          <%= f.label :_destroy, t('automated_tests.remove_file') %> 
-          <%= f.check_box :_destroy %>
+          <%= f.label :_destroy, t('automated_tests.remove_file') %>
+          <%= f.check_box :_destroy, :class => "remove_test_script_file" %>
 
         </div>
       </fieldset>
 
     <% else %>
       <%# Test Support File does not exist -- it is new %>
-        <%= raw t('automated_tests.upload_required') %>
-        <%= f.file_field :file_name, :size=>"30" %>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+        <%= t('automated_tests.new_test_support_file') %>
+        <br/>
+        <%= t('automated_tests.upload_required').html_safe %>
+        <%= file_field_tag :file_name, :size=>"30", :class => 'upload_file'%>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Please change the field name to `new_support_file` (file_name is not very helpful).
Also, please get rid of the additional new lines below.